tests/foreman/cli/test_remoteexecution.py

```diff
@@ -367,3 +424,137 @@ def test_positive_run_scheduled_job_template(self):
+            invocation_command = make_job_invocation({
+                'job-template': 'Run Command - SSH Default',
+                'inputs': 'command="ls"',
+                'search-query': "name ~ {0} or name ~ {1}".format(
+                    self.client.hostname, client2.hostname),
+            })
+            # collect output messages from clients
+            output_msgs = []
+            for vm in self.client, client2:
+                output_msgs.append('host output from {0}: {1}'.format(
+                        vm.hostname,
+                        ' '.join(JobInvocation.get_output({
+                            'id': invocation_command[u'id'],
+                            'host': vm.hostname})
+                        )
+                    )
+                )
+            self.assertEqual(invocation_command['success'], u'1', output_msgs)
+
+    @tier3
+    def test_positive_install_multiple_packages_with_a_job(self):
+        """Run job to install several packages on host
+
+        :id: 1cf2709e-e6cd-46c9-a7b7-c2e542c0e943
+
+        :expectedresults: Verify the packages were successfully installed
+            on host
+        """
+        # Create a custom repo
+        setup_org_for_a_custom_repo({
+            u'url': FAKE_0_YUM_REPO,
+            u'organization-id': self.org['id'],
+            u'content-view-id': self.content_view['id'],
+            u'lifecycle-environment-id': self.env['id'],
+            u'activationkey-id': self.activation_key['id'],
+        })
+        invocation_command = make_job_invocation({
+            'job-template': 'Install Package - Katello SSH Default',
+            'inputs': 'package={0} {1} {2}'.format(
+                "cow", "dog", "lion"),
+            'search-query': "name ~ {0}".format(self.client.hostname),
+        })
+        self.assertEqual(
+                invocation_command['success'],
+                u'1',
+                'host output: {0}'.format(
+                    ' '.join(JobInvocation.get_output({
+                        'id': invocation_command[u'id'],
+                        'host': self.client.hostname})
+                    )
+                )
+            )
+        result = ssh.command(
+                "rpm -q {0}".format("cow dog lion"),
+                hostname=self.client.hostname
+                )
+        self.assertEqual(result.return_code, 0)
+
+    @tier3
+    def test_positive_run_recurring_job_with_max_iterations(self):
+        """Run default job template multiple times with max iteration
+
+        :id: 37fb7d77-dbb1-45ae-8bd7-c70be7f6d949
+
+        :expectedresults: Verify the job was run not more than the specified
+            number of times.
+        """
+        invocation_command = make_job_invocation({
+            'job-template': 'Run Command - SSH Default',
+            'inputs': 'command="ls"',
+            'search-query': "name ~ {0}".format(self.client.hostname),
+            'cron-line': '* * * * *',  # every minute
```

Comment:
afaik, you can't get below 1 minute in crontab, I think what you propose is actually every 6th minute